src/View/Helper/RssHelper.php

```diff
@@ -327,6 +327,9 @@ public function elem($name, $attrib = [], $content = null, $endTag = true)
                 case 'atom':
                     $xml .= ' xmlns:atom="http://www.w3.org/2005/Atom"';
                     break;
+                default:
+                    $bareName = $name;
```

Comment:
I think you'd need to update/set the xmlns for this attribute in the document. That should appease libxml.